autopilot/autopilot.go

```diff
@@ -68,6 +98,8 @@ func New(store Store, bus Bus, worker Worker) (*Autopilot, error) {
 		store:  store,
 		bus:    bus,
 		worker: worker,
+
+		stopChan: make(chan struct{}),
```

Comment:
Could potentially be a context from which we can derive more children for subroutines?

Reply:
contexts are supposed to be request-scoped, aren't they? :P

Reply:
resolving this for now